lazy_tensor_core/lazy_tensor_core/csrc/lazy_view.cpp

```diff
@@ -74,14 +71,14 @@ torch::lazy::Value ApplyUpdate(torch::lazy::Value ir_value,
     const ViewInfo& view_info = update_data.view_infos[i - 1];
     switch (view_info.view_type) {
       case ViewInfo::Type::kSelect:
-        result = torch::lazy::MakeNode<ir::ops::Unselect>(
+        result = torch::lazy::MakeNode<ir::ops::SelectReverse>(
```

Comment:
this and GenericSliceReverse - just renaming the same things, but for more clarity?

Reply:
To make them more consistent across view ops. Before there are `AsStridedViewUpdate`, `DiagonalViewUpdate`, `UpdateSlice` and `Unselect`, and I have changed them into `*Reverse`.